bin/src/command/mod.rs

```diff
@@ -148,7 +123,7 @@ impl std::fmt::Display for Success {
             Self::ClientClose(id) => write!(f, "Close client: {id}"),
             Self::ClientNew(id) => write!(f, "New client successfully added: {id}"),
             Self::DumpState(_) => write!(f, "Successfully gathered state from the main process"),
-            Self::HandledClientRequest => write!(f, "Successfully handled the client request"),
+            Self::HandledClientOrder => write!(f, "Successfully handled the client request"),
```

Comment:
Same I prefer `Request` instead of `Order`. We should discuss about that :)

Reply:
Request is way better, I'll keep that !